pkg/connector/client.go

```diff
@@ -144,13 +181,17 @@ func (m *MetaClient) GetCapabilities(ctx context.Context, portal *bridgev2.Porta
 }
 
 // GetChatInfo implements bridgev2.NetworkAPI.
-func (m *MetaClient) GetChatInfo(ctx context.Context, portal *bridgev2.Portal) (*bridgev2.PortalInfo, error) {
-	panic("unimplemented")
+func (m *MetaClient) GetChatInfo(ctx context.Context, portal *bridgev2.Portal) (*bridgev2.ChatInfo, error) {
+	return &bridgev2.ChatInfo{
+		Name:         &[]string{"test"}[0],
+		IsSpace:      &[]bool{false}[0],
+		IsDirectChat: &[]bool{true}[0],
```

Comment:
btw there's a ptr util for making pointers https://pkg.go.dev/go.mau.fi/util/ptr / https://github.com/mautrix/go-util/blob/main/ptr/ptr.go

e.g. `ptr.Ptr(true)`